pages/about.tsx

```diff
@@ -19,6 +19,8 @@ const AboutPage: React.SFC<IProps> = ({ role, user_id }) => (
 (AboutPage as any).getInitialProps = async ({
     query: { role, user_id },
 }: IContext): Promise<IProps> => {
+    role = role as string | undefined;
+    user_id = user_id as string | undefined;
```

Comment:
Is it the good solution to cast these var? If it is, maybe it would be better to cast like that: return {role, user_id} as IProps ?

Reply:
And do you think we need to give an error when role and/or user_id are wrong on this page? Because we don't rly need them here, it's just to don't lose them on the url when we go to an other page

Reply:
```js
// if (foo === undefined) {
if (typeof foo !== "string") {
    throw new Error();
}
```